src/components/SelectInput/SelectInput.test.jsx

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import Select from 'react-select';
+import '@babel/polyfill';
+import 'mutationobserver-shim';
+import {
+  render,
+  fireEvent,
+  screen,
+} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import selectEvent from 'react-select-event';
+import SelectInput from './SelectInput';
+
+const selectOptions = [
+  { value: 'chocolate', label: 'Chocolate' },
+  { value: 'strawberry', label: 'Strawberry' },
+  { value: 'vanilla', label: 'Vanilla' },
+];
+
+describe('SelectInput', () => {
+  describe('Props Validation', () => {
+    test('Throws error if required prop "id" is not supplied to component', () => {
+      console.error = jest.fn(); // eslint-disable-line no-console
+      render(<SelectInput onChange={jest.fn()} options={selectOptions} />);
+      expect(console.error).toHaveBeenCalledTimes(1); // eslint-disable-line no-console
+      expect(console.error.mock.calls[0][0]) // eslint-disable-line no-console
+        .toContain('Failed prop type: The prop `id`');
+    });
+
+    test('Throws error if required prop "onChange" is not supplied to component', () => {
+      console.error = jest.fn(); // eslint-disable-line no-console
+      render(<SelectInput id="testId" options={selectOptions} />);
+      expect(console.error).toHaveBeenCalledTimes(1); // eslint-disable-line no-console
+      expect(console.error.mock.calls[0][0]) // eslint-disable-line no-console
+        .toContain('Failed prop type: The prop `onChange`');
+    });
+
+    test('Throws error if required prop "options" is not supplied to component', () => {
+      console.error = jest.fn(); // eslint-disable-line no-console
+      render(<SelectInput id="testId" onChange={jest.fn()} />);
+      expect(console.error).toHaveBeenCalledTimes(1); // eslint-disable-line no-console
+      expect(console.error.mock.calls[0][0]) // eslint-disable-line no-console
+        .toContain('Failed prop type: The prop `options`');
+    });
+  });
+
+  describe('Callback Handling', () => {
+    // test('it fires onChange callback on change', () => {
```

Comment:
I couldn't get this test to work. However, `onChange` is the most important callback. Could someone have a look?